app/serializers/location_free_spaces_serializer.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class LocationFreeSpacesSerializer
```

Comment:
What's to stop us reusing the existing locations serializer and adding a meta section to that?

Reply:
I explored this and there are a couple of issues - firstly meta support isn't optional. There are some outstanding issues and PR's on the serializer repo but the maintainer seems unresponsive to adding support for that. So even if the `meta` block returns `nil` then a `meta` key is added to every serialized location which I don't really want to introduce as a change. Secondly we don't care about associated suppliers so I don't want to involve extra queries for that relationship. Thirdly this only has sparse attributes for title since that's all that's required by the front end. TL:DR; this is an optimised serialiser designed to be fast and to deliver just the essential data required by the front end for this specific use case, without introducing breaking changes elsewhere.

Reply:
Clear. Thanks